Source/JavaScriptCore/runtime/ModuleProgramExecutable.cpp

```diff
@@ -73,9 +73,12 @@ UnlinkedModuleProgramCodeBlock* ModuleProgramExecutable::getUnlinkedCodeBlock(JS
 ModuleProgramExecutable* ModuleProgramExecutable::create(JSGlobalObject* globalObject, const SourceCode& source)
```

Comment:
We would pike to rename it to tryCreate when it can fail.

Reply:
updated